Lib/email/_header_value_parser.py

```diff
@@ -1039,7 +1039,7 @@ def get_encoded_word(value):
         raise errors.HeaderParseError(
             "expected encoded word but found {}".format(value))
     remstr = ''.join(remainder)
-    if len(remstr) > 1 and remstr[0] in hexdigits and remstr[1] in hexdigits:
+    if len(remstr) > 1 and remstr[0] in hexdigits and remstr[1] in hexdigits and tok.count('?') < 2:
```

Comment:
Probably needs to be split on multiple line for pep8.

Reply:
Done.

Is there a way to check for this automatically?